### PR TITLE
Add code for generating a generic commit and sending summary as mail

### PR DESCRIPTION
**Pull request by viktordick**

### CODE REVIEW

File: perfact/zodbsync/commands/record.py [hunk 1]
```diff
@@ -39,6 +34,41 @@ def add_args(parser):
...
+        """
+        Do a commit of all unstaged changes and optionally send an email with a
+        summary.
+        """
+        commit_message = self.config["commit_message"]
+        self.gitcmd_run('add', '.')
+        try:
+            self.gitcmd_run('commit', '-m', commit_message)
+        except subprocess.CalledProcessError:
+            # Nothing to commit
```
Comment (gandie) [inline perfact/zodbsync/commands/record.py:47]: Can we safely assume this always means `Nothing to commit` ?

**Review by gandie — COMMENTED** [review]
One question i'd like to clarify before approving

File: perfact/zodbsync/commands/record.py (hunk 1 above)
Comment (viktordick) [reply perfact/zodbsync/commands/record.py:47]: Probably not, although the old code in `python-perfact` did exactly that.
We can probably do better, but this is not a function any user calls explicitly, it is usually called from cron entries with `>/dev/null 2>&1` so if the commit is unsuccessful (for example, someone commited as root and others can now no longer commit), we will not notice, whatever we do - except maybe if we send an email.

Comment (gandie) [reply perfact/zodbsync/commands/record.py:47]: Alrighty, this will do for now! :+1:

**Review by gandie — APPROVED** [review]

**Pull request merged**